boards/imxrt1050-evkb/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "imxrt1050-evkb"
+version = "0.1.0"
+authors = ["Tock Project Developers <tock-dev@googlegroups.com>"]
+build = "build.rs"
+edition = "2018"
+
+[dependencies]
+components = { path = "../components" }
+cortexm7 = { path = "../../arch/cortex-m7" }
+capsules = { path = "../../capsules" }
+kernel = { path = "../../kernel" }
+imxrt1050 = { path = "../../chips/imxrt1050" }
+
+[build-dependencies]
+cc = "1.0.25"
```

Comment:
Why is this build dependency needed? Generally, Tock has a policy against external dependencies, which I think applies to build dependencies as well.

Reply:
Oh, i used it in the beginning when i compiled the boot header. Now it is no longer needed, so I deleted it.